### PR TITLE
Add FiNER dataset

### PR DESCRIPTION
**Pull request by stefan-it**
Hi,

this PR adds "A Finnish News Corpus for Named Entity Recognition" as new `finer` dataset.

The dataset is described in [this paper](https://arxiv.org/abs/1908.04212). The data is publicly available in [this GitHub](https://github.com/mpsilfve/finer-data).

Notice: they provide two testsets. The additional test dataset taken from Wikipedia is named as "test_wikipedia" split.

### CODE REVIEW

File: datasets/finer/finer.py [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+    def __init__(self, **kwargs):
+        """BuilderConfig for FiNER dataset.
+
+        Args:
+          **kwargs: keyword arguments forwarded to super.
+        """
+        super(FiNERConfig, self).__init__(**kwargs)
+
+
+class FiNER(datasets.GeneratorBasedBuilder):
```
Comment (lhoestq) [inline datasets/finer/finer.py:66]: Could you rename it Finer ? We're using camel case for class names.
Same for the config

File: datasets/finer/finer.py [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+    ]
+
+    def _info(self):
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=datasets.Features(
+                {
+                    "id": datasets.Value("string"),
+                    "tokens": datasets.Sequence(datasets.Value("string")),
+                    "ner_tags": datasets.Sequence(
```
Comment (lhoestq) [inline datasets/finer/finer.py:84]: perfect !

**Review by lhoestq — COMMENTED** [review]
That's really cool thank you !

Could you also add a dataset card ?
You can find a template here : https://github.com/huggingface/datasets/blob/master/templates/README.md

**Comment by yjernite** [comment]
> That's really cool thank you !
> 
> Could you also add a dataset card ?
> You can find a template here : https://github.com/huggingface/datasets/blob/master/templates/README.md

The full information for adding a dataset card can be found here :) 
https://github.com/huggingface/datasets/blob/master/ADD_NEW_DATASET.md#manually-tag-the-dataset-and-write-the-dataset-card

**Comment by stefan-it** [comment]
Thanks your suggestions! I've fixed them, and currently working on the dataset card!

**Comment by stefan-it** [comment]
@yjernite and @lhoestq I will add the dataset card a bit later in a separate PR if that's ok for you!

**Comment by thomwolf** [comment]
Yes I want to re-emphasize if it was not clear that dataset cards are optional for the sprint. 

Only the tags are required for merging a datasets.

Please try to enforce this rule as well @lhoestq and @yjernite

**Comment by lhoestq** [comment]
Yes @stefan-it if you could just add the tags (the yaml part at the top of the dataset card) that'd be perfect :)

**Comment by stefan-it** [comment]
Oh, sorry, will add them now!

**Comment by stefan-it** [comment]
Initial README file is now added :)

File: datasets/finer/README.md [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+multilinguality:
+- monolingual
+size_categories:
+- 1K<n<10K
+source_datasets:
+- original
+task_categories:
+- structure-prediction
+task_ids:
+- named-entity-recognition
```
Comment (lhoestq) [inline datasets/finer/README.md:19]:
```suggestion
---
annotations_creators:
- expert-generated
language_creators:
- other
languages:
- fi
licenses:
- mit
multilinguality:
- monolingual
size_categories:
- 1K<n<10K
source_datasets:
- original
task_categories:
- structure-prediction
task_ids:
- named-entity-recognition
---
```

**Review by lhoestq — APPROVED** [review]
Thanks !

**Comment by lhoestq** [comment]
the `RemoteDatasetTest ` errors in the CI are fixed on master so it's fine

**Comment by lhoestq** [comment]
merging since the CI is fixed on master

**Pull request merged**